ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -98,14 +100,22 @@ private fun Template2MainContent(state: PaywallViewState.Loaded, viewModel: Payw
 private fun IconImage(uri: Uri?) {
     uri?.let {
         Column(modifier = Modifier.widthIn(max = Template2UIConstants.maxIconWidth)) {
-            RemoteImage(
-                urlString = uri.toString(),
-                modifier = Modifier
-                    .aspectRatio(ratio = 1f)
-                    .widthIn(max = Template2UIConstants.maxIconWidth)
-                    .clip(RoundedCornerShape(Template2UIConstants.iconCornerRadius)),
-                contentScale = ContentScale.Crop,
-            )
+            // TODO-PAYWALLS: test this
+            if (uri.toString().startsWith("android.resource://")) {
+                Image(
+                    painter = painterResource(id = uri.toString().substringAfterLast("/").toInt()),
```

Comment:
So just to check, this will convert the string `R.drawable.background` to the correct resource int?

Reply:
I've removed this since we need to show the app icon here, not the background 🤦